app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def route_active_class?(test_path)
+    return 'active' if request.path == test_path
+
+    ''
```

Comment:
What are those `''`? 😄

Reply:
That's a default return, seems to be Ruby style 🤷